kit.yml

```diff
@@ -474,23 +478,6 @@ params:
       description: |
         This is the password that the `bbs` nodes will use to connect to their database with
 
-    - ask: "What hostname/IP is the Routing API Database accessible at?"
-      description: |
-        This is the hostname that the `routing_api` will use to connect to its database
-      param: routingdb_host
-    - ask: "What port is the Routing API Database accessible at?"
-      description: |
-        This is the port that the `routing_api` will use to connect to its database
-      param: routingdb_port
-    - ask: "What is the username used to connect to the Routing API Database?"
-      description: |
-        This is the username that the `routing_api` will use to connect to its database
-      vault: routingdb:user
-    - ask: "What is the password used to connect to the Routing API Database?"
-      vault: routingdb:password
-      description: |
-        This is the password that the `routing_api` will use to connect to its database
-
```

Comment:
I think moving this out of the main kit into the tcp-router may break non-tcp-routing deployments, as the routing-api is always deployed on the cloud controller, and it uses these credentials